delivery_transsmart/models/booking_profile.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Therp BV <https://therp.nl>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+from openerp.exceptions import ValidationError
+
+
+class BookingProfile(models.Model):
+    _name = 'booking.profile'
```

Comment:
@daramousk Shouldn't this part of code in another module or be renamed with 'transmart...' ? If it could be a generic behaviour, that should be interesting to have it available elsewhere without pulling transsmart module.